tests/lib/utils/ember-test.js

```diff
@@ -492,13 +569,58 @@ describe('isComputedProp', () => {
 
 describe('isObserverProp', () => {
   let node;
+  let context;
 
   it("should check if it's an observer prop", () => {
-    node = parse('observer()');
+    context = new FauxContext(`
```

Comment:
What do you think about splitting each of these different "types" into their one test cases? That would help make the test output more clear about where something is going wrong, if one of these cases no longer worked right.

Reply:
Yeah, it would generally be nice to have more, smaller unit tests.

Reply:
Sounds good